02-Homework/script.js

```diff
@@ -0,0 +1,56 @@
+// Assignment Code
+var generateBtn = document.querySelector('#generate');
+
+function generatePassword() {
+  // PLEASE ADD ALL YOUR CODE HERE
+}
+alert ("text");
+var text
+var intiger
+if ( text = intiger) {
+"check is within range"
+} else {
+  alert and break
```

Comment:
Where have you got this code from? What do you expect `alert` and `break` to do?